features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -78,15 +83,28 @@ internal class Base64Serializer private constructor(
     // region private
 
     @WorkerThread
-    private fun serialiseBitmap(
+    private fun serializeBitmap(
         drawable: Drawable,
+        displayMetrics: DisplayMetrics,
         bitmap: Bitmap,
         shouldCacheBitmap: Boolean,
         imageWireframe: MobileSegment.Wireframe.ImageWireframe,
-        base64SerializerCallback: Base64SerializerCallback?
+        base64SerializerCallback: Base64SerializerCallback,
+        isFailover: Boolean
     ) {
-        val base64String = convertBmpToBase64(drawable, bitmap, shouldCacheBitmap)
-        finalizeRecordedDataItem(base64String, imageWireframe, base64SerializerCallback)
+        val base64String = convertBmpToBase64(
```

Comment:
Let's try to use a common lingvo and use `bitmap` everywhere

```suggestion
        val base64String = convertBitmapToBase64(
```